composer.json

```diff
@@ -86,6 +86,7 @@
         "hautelook/alice-bundle": "^2.8",
         "phpspec/prophecy-phpunit": "^2.0",
         "phpstan/phpstan": "^1.3",
+        "phpunit/phpunit": "^9.5",
```

Comment:
We already have this installed through the dev requirement`symfony/phpunit-bridge` (https://symfony.com/doc/5.4/components/phpunit_bridge.html), so is this necessary?

Reply:
Not really, but https://symfony.com/doc/current/testing.html#the-phpunit-testing-framework recommends (i.e. tells us) to install `symfony/test-pack` which requires `phpunit/phpunit` (cf. https://github.com/symfony/test-pack/blob/main/composer.json).